x/staking/types/validator_test.go

```diff
@@ -317,3 +319,18 @@ func TestValidatorSetInitialCommission(t *testing.T) {
 		}
 	}
 }
+
+func TestValidatorMarshalYAML(t *testing.T) {
+	validator := NewValidator(valAddr1, pk1, Description{})
+	bechifiedPub, err := sdk.Bech32ifyConsPub(validator.ConsPubKey)
+	require.NoError(t, err)
+	bs, err := yaml.Marshal(validator)
+	require.NoError(t, err)
+	want := fmt.Sprintf("|\n  operatoraddress: %s\n  conspubkey" +
```

Comment:
mmm could we use back ticks ` instead of double " ? This is hard to read

Reply:
done